src/main/java/software/amazon/smithy/lsp/ext/SmithyProject.java

```diff
@@ -172,7 +172,13 @@ private static Either<Exception, SmithyProject> load(
             return Either.forLeft(model.getLeft());
         } else {
             model.getRight().getValidationEvents().forEach(LspLog::println);
-            return Either.forRight(new SmithyProject(imports, smithyFiles, externalJars, root, model.getRight()));
+
+            try {
+                SmithyProject p = new SmithyProject(imports, smithyFiles, externalJars, root, model.getRight());
```

Comment:
**thought:** The issues I've been seeing are all during this constructor call, so there's no reason to assume it doesn't throw. Catching here means that `createProject` will send a message to the client. Potentially it'd be a better place for the try/catch? Or should we do it in both, just for future generations?